sdk/communication/azure-communication-sms/README.md

```diff
@@ -15,6 +15,38 @@ Azure Communication SMS is used to send simple text messages.
 - A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
 
 ### Include the package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```

Comment:
> Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.